### PR TITLE
NEXTEUROPA-12401

### PR DESCRIPTION
**Pull request by cyberwolf**

### CODE REVIEW

File: tests/features/embedded_image.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  And I press "Next"
+  Then the media browser closes
+  And I see the "Flickr set" preview
+  When I press "Save"
+  Then I should see the success message "Media Gallery <title> has been created."
+
+  Examples:
+  | title              | url                                                |
+  | Old Flickr set URL | https://www.flickr.com/photos/junku/sets/303691/   |
+  | New Flickr set URL | https://www.flickr.com/photos/junku/albums/303691/ |
```
Comment (voidtek) [inline tests/features/embedded_image.feature]: It's better to use an example associated with the EC. (like this user: https://www.flickr.com/photos/69583224@N05/)

**Comment by voidtek** [comment]
:+1:

**Comment by Fefaine** [comment]
:+1:

**Comment by Fefaine** [comment]
:smile_cat:

**Pull request merged**